packages/ccda/src/fhir-to-ccda.ts

```diff
@@ -1248,8 +1257,8 @@ class FhirToCcdaConverter {
       id: this.mapIdentifiers(observation.id, observation.identifier) as CcdaId[],
       code: mapCodeableConceptToCcdaCode(observation.code),
       statusCode: { '@_code': 'completed' },
-      value: this.mapObservationValue(observation),
       effectiveTime: [{ '@_value': mapFhirToCcdaDateTime(observation.effectiveDateTime) }],
+      value: this.mapObservationValue(observation),
```

Comment:
Order is important in the XML output 😢 

When this matures a little more, I plan to refactor the tests to look at exact string output rather than XML tree objects, which are more loose with objects.